### ISSUE TITLE
crash when clicking on deleted post

### ISSUE THREAD

**Issue by Quantenzitrone** [Docile-Alligator/Infinity-For-Reddit#1353]
## Scoop (stack trace)
```
FATAL EXCEPTION: main
Process: ml.docilealligator.infinityforreddit, PID: 5051
java.lang.RuntimeException: Unable to start activity ComponentInfo{ml.docilealligator.infinityforreddit/ml.docilealligator.infinityforreddit.activities.LinkResolverActivity}: android.view.InflateException: Binary XML file line #27: Binary XML file line #27: Error inflating class TextView
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2819)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2897)
	at android.app.ActivityThread.-wrap11(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1623)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:164)
	at android.app.ActivityThread.main(ActivityThread.java:6548)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:857)
Caused by: android.view.InflateException: Binary XML file line #27: Binary XML file line #27: Error inflating class TextView
Caused by: android.view.InflateException: Binary XML file line #27: Error inflating class TextView
Caused by: java.lang.UnsupportedOperationException: Failed to resolve attribute at index 6: TypedValue{t=0x2/d=0x101009b a=1}
	at android.content.res.TypedArray.getColorStateList(TypedArray.java:538)
	at android.widget.TextView.<init>(TextView.java:914)
	at android.widget.TextView.<init>(TextView.java:828)
	at androidx.appcompat.widget.AppCompatTextView.<init>(AppCompatTextView.java:108)
	at com.google.android.material.textview.MaterialTextView.<init>(MaterialTextView.java:93)
	at com.google.android.material.textview.MaterialTextView.<init>(MaterialTextView.java:88)
	at com.google.android.material.textview.MaterialTextView.<init>(MaterialTextView.java:83)
	at com.google.android.material.theme.MaterialComponentsViewInflater.createTextView(MaterialComponentsViewInflater.java:61)
	at androidx.appcompat.app.AppCompatViewInflater.createView(AppCompatViewInflater.java:121)
	at androidx.appcompat.app.AppCompatDelegateImpl.createView(AppCompatDelegateImpl.java:1565)
	at androidx.appcompat.app.AppCompatDelegateImpl.onCreateView(AppCompatDelegateImpl.java:1616)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:772)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:730)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:863)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
	at android.widget.Toast.makeText(Toast.java:281)
	at android.widget.Toast.makeText(Toast.java:267)
	at android.widget.Toast.makeText(Toast.java:304)
	at ml.docilealligator.infinityforreddit.activities.LinkResolverActivity.onCreate(LinkResolverActivity.java:96)
	at android.app.Activity.performCreate(Activity.java:7009)
	at android.app.Activity.performCreate(Activity.java:7000)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1214)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2772)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2897)
	at android.app.ActivityThread.-wrap11(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1623)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:164)
	at android.app.ActivityThread.main(ActivityThread.java:6548)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:857)
```

## Reproduce
https://www.reddit.com/r/HentaiBullying/comments/hv870v/aww_how_mean_my_hand_slipped/
- open in infinity
- click on the post

## Phone info
- Model: SHIFT6m
- Android version: 8.1
- ROM: ShiftOS-L (degoogled)

**Comment by Wladefant** [comment]
thank you. Next time please fill out the bug report form

**Comment by Quantenzitrone** [comment]
Sorry, i did this with octodroid, something must have bugged out because i didn't get the bug report form.

**Comment by seirdotexe** [comment]
> Sorry, i did this with octodroid, something must have bugged out because i didn't get the bug report form.

Both Octodroid and the official Github app don't support the bug report form. The official Github app also sends you to the browser to fill it in there. This is either a bug in the form or not supported?

**Comment by Wladefant** [comment]
@seirdotexe not supported by the app, but github mobile does not let you create a blank page

**Comment by Docile-Alligator** [comment]
Could not reproduce this on v5.4.0.

**Comment by Quantenzitrone** [comment]
i can reproduce this in v5.4.0

**Comment by Wladefant** [comment]
@Quantenzitrone what do you mean by broken? Do you mean deleted? Does the crash still happen after the post got deleted?

**Comment by Quantenzitrone** [comment]
yep deleted, but it wasn't obvious that it is deleted from infinity, i had to open it in the browser

**Comment by Wladefant** [comment]
For me it is showing that it is deleted but it still says that a link is there. When you copy the link it seems like it is an empty link. So thus is definitely a bug

**Comment by Docile-Alligator** [comment]
Fixed in v5.4.1.
